lib/Echidna/Solidity.hs

```diff
@@ -80,10 +81,13 @@ instance Show SolException where
     OnlyTests            -> "Only tests and no public functions found in ABI"
     (ConstructorArgs s)  -> "Constructor arguments are required: " ++ s
     NoCryticCompile      -> "crytic-compile not installed or not found in PATH. To install it, run:\n   pip install crytic-compile"
+    InvalidMethodFilters -> "Filtering methods whitelisting or blacklisting cannot result empty"
```

Comment:
we can make this a bit more descriptive, see slack chat

Reply:
Done!